helm-charts/splunk-otel-collector/templates/config/_common.tpl

```diff
@@ -153,12 +155,28 @@ Filter logs processor
 # Drop logs coming from pods and namespaces with splunk.com/exclude annotation.
 filter/logs:
   logs:
-    exclude:
+    {{ .Values.logsCollection.containers.useSplunkIncludeAnnotation | ternary "include" "exclude" }}:
+      match_type: strict
       resource_attributes:
-        - key: splunk.com/exclude
+        - key: splunk.com/{{ .Values.logsCollection.containers.useSplunkIncludeAnnotation | ternary "include" "exclude" }}
           value: "true"
 {{- end }}
 
+{{/*
+Filter Attributes Function
+*/}}
+{{- define "splunk-otel-collector.filterAttr" -}}
+{{- if .Values.logsCollection.containers.useSplunkIncludeAnnotation }}
+- key: splunk.com/include
+  tag_name: splunk.com/include
+  from: pod
```

Comment:
My idea was to return only `splunk.com/include` or `splunk.com/exclude` in this helper and update _all_ the places where `splunk.com/exclude` currently is being used:
- resource/logs delete action
- k8sattributes processor
- filter/logs processor

For example
```yaml
resource/logs:
  attributes:
    - key: {{ include "splunk-otel-collector.filterAttr" . }}
       action: delete
```

WDYT? Can we make it this way? And here I'm assuming we apply it consistently to pods and namespaces. I still don't understand why we have to introduce this inconsistency just because a customer wants it to be scraped from pods for now. I believe having it as a consistent config for both pods and namespaces is what can be expected by others and will not introduce any problems to the customer who asked for this feature initially.